db/seeds.rb

```diff
@@ -0,0 +1,83 @@
+# This file should contain all the record creation needed to seed the database with its default values.
```

Comment:
This seeds file ingests `restaurants_times.csv` and adds each `Restaurant` to the database.